packages/gatsby-theme-newrelic/index.js

```diff
@@ -6,3 +6,8 @@ export { default as Icon } from './src/components/Icon';
 export { default as MDXCodeBlock } from './src/components/MDXCodeBlock';
 export { default as NewRelicLogo } from './src/components/NewRelicLogo';
 export { default as HamburgerMenu } from './src/components/HamburgerMenu';
+export { default as SearchInput } from './src/components/SearchInput';
+export { default as formatCode } from './src/utils/formatCode';
```

Comment:
```suggestion
export { default as SearchInput } from './src/components/SearchInput';

export { default as formatCode } from './src/utils/formatCode';
```

Reply:
@jerelmiller yes! sorry i was just about to recommit with the changes, I also found a few stylistic bugs that I am now committing.